libraries/android/castor-tokens/build.gradle

```diff
@@ -0,0 +1,35 @@
+plugins {
+    id "com.android.library"
+    id "kotlin-android"
+}
+
+android {
+    def versionFile = file("./version.json")
+    def parsedVersionText = new groovy.json.JsonSlurper().parseText(versionFile.text)
+
+    compileSdkVersion 30
+    buildToolsVersion "30.0.3"
+
+    defaultConfig {
+        minSdkVersion 24
+        targetSdkVersion 30
+        versionCode parsedVersionText.versionCode
+        versionName parsedVersionText.versionName
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
```

Comment:
`minifyEnabled` (true) turn on proguard/R8 which obfuscate and remove unused codes so I'd suggest true for `minifyEnabled`